content/blog/2022-07-13-Galaxy-Wellbeing-Blog-4/index.md

```diff
@@ -0,0 +1,65 @@
+
+# Time-zone difference
+
+Open-source projects allow anyone from different parts of the world to contribute freely and work together in a remote setting. Having a global team has its advantages in terms of flexibility, culture, and such, but it can sometimes be challenging to accommodate the different time zones of everyone on the team. The problems can lead to working after hours or extra hours, having meetings very early in the morning or late in the afternoon or night, not being able to communicate effectively across teams, decreased productivity, poor collaboration, and such.
+
+## What can be done to lessen the challenges?
+
+**Set boundaries**: Everyone should set boundaries by indicating working hours or times they will be available to meet or work together.
+
+**Be flexible to irregular working hours**: Sometimes it might be hard to set working hours for open-source projects, since some people usually work when they get free time. In cases like this, have a communication channel where you can communicate asynchronously about the work or plans. Don’t expect an immediate response; let them get back to you when they can.
+
+**Be mindful of other people’s working hours**: Respect others working hours and limit yourself from contacting them or scheduling a meeting outside that hour. Finding overlapping working hours for meetings can be difficult sometimes. In times like that, try having two meetings at different reasonable times so people could be able to attend one that suits their time-zone as well as record meetings and share notes. Always communicate in advance for any plans that might require working after available hours and take NO for an answer. Sometimes it’s hard for people to say no for different reasons, so generally, it's best to refrain from asking to meet or work outside available hours unless it’s absolutely necessary. When doing so, don’t just inform people of the plan; always ask if it’s possible to meet or work. Don’t push others to work in your time by insisting or asking for unnecessary favors. Always check the other person's time zone before doing anything, whether it's sending emails or scheduling a meeting. 
+
+When sending emails, if the person you are working with has a different working hour than you, consider:
+* Scheduling your email to be sent at the receiver's working hour
+* Including a note stating your email is sent during your working hours mentioning your working hours in UTC. 
+
+**Create a collaborative asynchronous workspace**: To work and collaborate effectively, create a space where you can share work, schedule meetings, and communicate across different time zones. This can be done by using project management apps such as [Trello](https://trello.com/en) or [ClickUp](https://clickup.com/) or shared calendars.  
```

Comment:
```suggestion
**Create a collaborative asynchronous workspace**: To work and collaborate effectively, create a space where you can share work, schedule meetings, and communicate across different time zones. This can be done by using project management apps such as [Trello](https://trello.com/en), [ClickUp](https://clickup.com/) or shared calendars.  
```